examples/next-chat/yarn.lock

```diff
@@ -0,0 +1,24 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@examples/chat@6.3.1":
+  version "6.3.1"
+  dependencies:
+    "@prisma/client" "^2.18.0"
+    "@trpc/client" "^6.3.1"
+    "@trpc/next" "^6.3.1"
+    "@trpc/react" "^6.3.1"
+    "@trpc/server" "^6.3.1"
+    jest "^26.6.3"
+    jest-playwright "^0.0.1"
+    jest-playwright-preset "^1.4.5"
+    next "^10.0.5"
+    playwright "^1.8.0"
+    react "^17.0.1"
+    react-dom "^17.0.1"
+    react-query "^3.6.0"
+    start-server-and-test "^1.12.0"
+    superjson "^1.5.2"
+    zod "^3.0.0"
+
```

Comment:
This isn't a "proper" lockfile unfortunately, it just regurgitates what's in `package.json`, doesn't contain hashes of the "current" version, etc

Reply:
That's a shame, using this yarn command to generate:
https://classic.yarnpkg.com/en/docs/cli/generate-lock-entry/

I'm thinking that maybe the example shouldn't part of yarn workspace, so we can generate the yarn lock for each.

I will try and give another go this week, hopefully :)